Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Dockerfile for cBioPortal
+FROM elementolab/jamm:1.0
+MAINTAINER Alexandros Sigaras <als2076@med.cornell.edu>
+#===========================#
+# Clone cBioPortal			#
+#===========================#
+#RUN git clone https://github.com/cBioPortal/cbioportal.git; \
+ADD / /cbioportal
+RUN	echo "export PORTAL_HOME=/cbioportal" >> /root/.bash_profile;
```

Comment:
I’d say both. Several commands in the scripts package, as well as `mvn test`, use the `PORTAL_HOME` variable.